File: wiz-event-mall-java/src/main/java/com/example/wizeventmalljava/domain/event/repository/EventRepository.java
package com.example.wizeventmalljava.domain.event.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import com.example.wizeventmalljava.domain.event.entity.Event;

@Repository
public interface EventRepository extends JpaRepository<Event, Long> {
}

File: wiz-event-mall/src/main/kotlin/com/example/wizeventmall/domain/event/repository/EventRepository.kt
package com.example.wizeventmall.domain.event.repository

import com.example.wizeventmall.domain.event.entity.Event
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface EventRepository: JpaRepository<Event, Long> {
}